force-app/main/default/classes/BDI_ContactService.cls

```diff
@@ -92,6 +121,26 @@ public with sharing class BDI_ContactService {
         set;
     }
 
+    private Map<String,String> mapDIHomeAddrToContact {
+        get {
+            mapDIHomeAddrToContact = new Map<String,String> {'Home_Street__c' => 'MailingStreet' ,
```

Comment:
Is there a risk of namespace issues here?

Reply:
Yeah let me take another look

Reply:
Updated these to dynamically apply namespace